scala/integration/intellij-bsp/resources/META-INF/scala-intellij-bsp-integration.xml

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <idea-plugin>
     <extensions defaultExtensionNs="org.jetbrains.bsp">
-        <scalaSdkExtension implementation="org.jetbrains.plugins.scala.project.bsp.ScalaSdkExtensionImpl"/>
+        <bspProjectOpenProcessorExtension
+            implementation="org.jetbrains.plugins.scala.bsp.flow.open.ScalaBspProjectOpenProcessorExtension" />
+        <bspProjectAwareExtension
+            implementation="org.jetbrains.plugins.scala.bsp.config.ScalaBspProjectAwareExtension" />
+        <buildTargetClassifierExtension
+            implementation="org.jetbrains.plugins.scala.bsp.extension.ScalaBuildTargetClassifier" />
+        <buildToolAssetsExtension
+            implementation="org.jetbrains.plugins.scala.bsp.assets.ScalaAssetsExtension" />
+        <scalaSdkExtension
+            implementation="org.jetbrains.plugins.scala.project.bsp.ScalaSdkExtensionImpl" />
+        <ConnectionDetailsProviderExtension
```

Comment:
typo in `ConnectionDetailsProviderExtension`.
it should be `connectionDetailsProviderExtension`.
Please double-check with the highlighting feature to make sure you are using the correct extension names.